src/main/resources/templates/profile.html

```diff
@@ -13,7 +13,7 @@
 
     <div class="row">
         <nav class="navbar navbar-expand-lg navbar-light bg-light" style="width:100%">
-            <a class="navbar-brand" href="#">Navbar</a>
+            <a class="navbar-brand" href="/">Amazin</a>
```

Comment:
Maybe "Amazin Bookstore" instead of just Amazin? Only if it works out in terms of spacing and whatnot

Reply:
I think it should work, I can change that